microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-105.md

```diff
@@ -0,0 +1,180 @@
+---
+title: What's new in DevTools (Microsoft Edge 105)
+description: "Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View. Improved reliability in Issues tool. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 09/01/2022
+---
+# What's New in DevTools (Microsoft Edge 105)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+<!-- ====================================================================== -->
+## Test your PWA's protocol handlers from the Application tool-->
+
+<!-- Rank: 1 -->
+<!-- Title: Test your PWA's protocol handlers from the Application tool  -->
+<!-- Subtitle: From the Manifest section of the Application tool, you can now provide custom protocols to launch your PWA. -->
+
+In Microsoft Edge 105, the Application tool now supports testing protocol handlers. Since Microsoft Edge 96, you've been able to define protocol handlers in the application manifest of your PWA. Now you can  test those protocols from the Application tool itself in DevTools if you have your PWA installed. 
+
+With DevTools open against your PWA:
+1. Open the Application tool.
+2. Expand the Manifest dropdown.
+3. Select the Protocol Handlers entry in the dropdown.
+4. Select the protocol you want to test from the dropdown.
+5. Enter in the URL or endpoint that you would like to test.
+6. Select **Test Protocol**.
+
+The Application tool will attempt to launch your PWA with the protocol and URL you specified. The browser asks for permission to open the application, and then prompts you to confirm that you want to handle the protocol and the app you'd like to launch. If you give permission, your app will open with the specified content.
+
+<!-- (how to configure the DevTools correctly, a website or CodePen that the writer can use, where to navigate in the DevTools UI for the screenshot, etc.)
+Refer to the attachment.
+
+To take your own screenshot:
+1. Open Microsoft Edge Beta, Dev, or Canary (just go to edge://version and ensure you're on version 105+)
+2. Navigate to URL protocol handler (protocol-handler.glitch.me).
+3. Install the PWA (see docs).
+4. Open DevTools > Application tool.
+5. Select the Manifest dropdown to expand it.
+6. Select the Protocol Handler entry in the dropdown.
+7 With the web+coffee:// protocol selected, enter into the textbox "americano".
+8 Take a screenshot. Draw red highlight boxes over the "Protocol handlers" entry in the Manifest dropdown and around the Protocol Handlers section in the panel. -->
+
+![1](./devtools-105-images/protocol-handler.png)
```

Comment:
put real alt text instead of "1"

Reply:
To improve this image, you could also add a red box around the protocol handlers section, and add some text in the textfield next to web+coffee://, to show users that's where they're supposed to be testing their protocols.

Reply:
So at the last minute it was decided not to use the text in the box because Americano might not always be there, but I added the red box around the textfield as that's smart.